src/main/java/io/vertx/core/eventbus/DeliveryOptions.java

```diff
@@ -188,4 +197,25 @@ private void checkHeaders() {
       headers = new CaseInsensitiveHeaders();
     }
   }
+
+  /**
+   * @return whether the message should be delivered to local consumers only
+   */
+  public boolean isLocalOnly() {
+    return localOnly;
+  }
+
+  /**
+   * Whether a message should be delivered to local consumers only. Defaults to {@code false}.
+   *
+   * <p>
+   * <strong>This option is effective in clustered mode only and does not apply to reply messages</strong>.
```

Comment:
I think you should have use <blink> instead

Reply:
lol